### PR TITLE
Snap 234 search client on edge

### PR DESCRIPTION
**Pull request by money**
https://osi-cwds.atlassian.net/browse/SNAP-234

Description:
This handles the issue with entering text in the search field on Edge.

### CODE REVIEW

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -35,6 +34,13 @@
   end
 end
 
+  Capybara.register_driver :selenium_ie do |app|
+    capabilities = Selenium::WebDriver::Remote::Capabilities.ie(
+      ignore_zoom_setting: true
+    )
+    Capybara::Selenium::Driver.new(app, browser: :ie)
```
Comment (NagaPradeepGajula-osi) [inline spec/spec_helper.rb]: The Zoom was a pain for IE setting. Nice finding and fix. 💯

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

File: spec/snapshot/state_sensitive_user_spec.rb [hunk 2]
```diff
@@ -43,12 +43,17 @@
           expect(page).to have_css('span.person', text: same_county_sensitive_not_sealed.full_name)
         end
       end
+    end
 
-      it 'should be able to attach different_county_sensitive_not_sealed client' do
-        search_client(label: 'Search for clients',
-                      query: different_county_sensitive_not_sealed.search_name)
-        find('strong.highlighted', text: different_county_sensitive_not_sealed.full_name).click
+    describe 'privilege revoked' do
```
Comment (wmitchell-cambria) [inline spec/snapshot/state_sensitive_user_spec.rb]: Is this change intentional? It looks like it undoes a recent PR to update these expectations, and it causes a duplicate `describe 'privilege revoked'` block.

**Comment by money** [comment]
Known issues:
https://github.com/teamcapybara/capybara/issues/1890
https://bugs.chromium.org/p/chromedriver/issues/detail?id=1771&q=sendkeys&sort=-id&colspec=ID%20Status%20Pri%20Owner%20Summary
